### DISCUSSION TITLE
UDMP 1.12.30 wont reinstall udm-iptv as it fails with configuration process

### DISCUSSION THREAD

**Discussion by Darksprout** [fabianishere/udm-iptv#190] [General]
Hello everyone

After the latest firmware upgrade to 1.12.30 on UDMP I cannot reinstall udm-iptv as it fails with a locked blue screen when it runs the config section. It wont let me progress the config the usual way, each time i press a key or click the mouse I get weird error codes.

Running a grep on the processes allows me to kill them, but I cannot then run dpkg-reconfigure -p medium udm-iptv because it isn't fully installed now.
Has anyone else re-run the installer on latest firmware? Are you aware of any updates needed with the new firmware @fabianishere 

Its run flawlessly for almost 12 months and now I get this :(
I hope someone can help as the family will notice the missing KPN TV very quickly
Screenshot below
Many thanks
<img width="1115" alt="Screenshot 2022-09-28 at 20 26 03" src="https://user-images.githubusercontent.com/111918676/192863351-2169de0d-42bd-4ce3-8695-65289b05316b.png">

**Comment by fabianishere** [comment]
You can try adding `DEBIAN_FRONTEND=readline` before the APT command to force it to use the text-mode installation.

**Comment by fabianishere** [comment]
Please try the updated installation command:

```bash
sh -c "$(curl https://raw.githubusercontent.com/fabianishere/udm-iptv/master/install.sh -sSf)"
```
